### ISSUE TITLE
spectrum scale: add performance options

### ISSUE THREAD

**Issue by wookietreiber** [idiv-biodiversity/stor-age#9]
There are approximately 1 million files in `/data/dir`, the node class **filer** has 7 nodes, each has at least 16 physical CPU cores:

```console
$ time mmapplypolicy /data/dir -P stor-age.policy -f stor-age -I defer -L 0
real    0m40.065s

$ $base -N filer
real    0m23.452s

$ $base -N filer -g /work/.policy/
real    0m13.636s

$ $base -N filer -g /work/.policy/ --choice-algorithm fast
real    0m12.505s

$ $base -N filer -g /work/.policy/ --choice-algorithm fast -a 16
real    0m12.514s
```

According to documentation, `--choice-algorithm` and `-a` sort of depend on `-N` **and** `-g`. So, we should at the very least use `-N` and `-g` to really boost performance.

I will do another benchmark using a directory with 180 million inodes to see if `--choice-algorithm` and `-a` have bigger impact there. At least with the 1 million files example they weren't that impactful.

**Comment by wookietreiber** [comment]
Here is the example with 180 million files:

```console
$ time mmapplypolicy /data/dir -P stor-age.policy -f stor-age -I defer -L 0
real    57m23.451s

$ $base -N filer -g /work/.policy/
real    16m36.549s

$ $base -N filer -g /work/.policy/ --choice-algorithm fast
real    15m6.883s

$ $base -N filer -g /work/.policy/ --choice-algorithm fast -a 16
real    14m4.068s
```

**Comment by wookietreiber** [comment]
Fixed in d573a29, cb0dcf5, 9498f92.
